src/Microsoft.Azure.WebJobs.Extensions.Kafka/Listeners/KafkaListener.cs

```diff
@@ -163,8 +163,8 @@ private ConsumerConfig GetConsumerConfiguration()
                 // Librdkafka debug options               
                 Debug = this.options.LibkafkaDebug,
 
-                // start from earliest if no checkpoint has been committed
-                AutoOffsetReset = AutoOffsetReset.Earliest,
+                // Librdkafka auto offset reset. By default set to Earliest.
+                AutoOffsetReset = this.options.AutoOffsetReset,
```

Comment:
by default if value is not found it should be earliest

Reply:
It's specified in the KafkaOptions
public AutoOffsetReset AutoOffsetReset { get; set; } = AutoOffsetReset.Earliest;